docs/datasync/conflict-resolution-intro.md

```diff
@@ -88,17 +88,14 @@ const {
   modelDefs,
   {
     db,
-    dataSyncConflictMap: {
-      Comment: {
-        enabled: true,
-        deltaTTL: 604800
-      }
+    conflictConfig: {
+      enabled: true
     }
   }
 );
 ```
 
-The `dataSyncConflictMap` argument is used to `enable` the server-side Conflict resolution feature for specific models, in addition to Delta Queries. When it is enabled, a `delta` table is maintained for each model which uses it. 
+The `conflictConfig` argument is used to either configure conflicts for all `@datasync` enabled models by directly setting global conflict parameters like `enabled` or configuring specific models via a `models` property. When conflicts are enabled for a specific model, a delta table is maintained for that model.
```

Comment:
I'd also add an example like https://github.com/aerogear/graphback/pull/1905/files#diff-36beda04046dab190232c0cf12a2999eR91 to show how one can provide custom model conflict config.

Reply:
What do you think?

Reply:
I have added an example here: https://github.com/aerogear/graphback/blob/c45fe8b3bc0907930fec23e418b7ed8a005d5d56/docs/datasync/conflict-resolution-strategies.md

Does that look okay?

Reply:
Ah right. That'll do. Maybe a link to that part will do?

Reply:
There's a link in the same file, under [this](https://github.com/aerogear/graphback/blob/c45fe8b3bc0907930fec23e418b7ed8a005d5d56/docs/datasync/conflict-resolution-intro.md#modify-the-template-to-use-createdatasyncapi) heading, near the end, would that be okay?

Reply:
My opinion is that it is kind of hidden by other docs about strategies. But I guess it is okay for now, as I do not have any other better idea. Let's see what other will say or we can revisit later if needed. Thanks!

Reply:
Yep. This is convoluted. I will refactor docs once we get something out there as they need some simplification aggregation of concepts you do normally and things that you do only in advanced cases.

Reply:
Sounds good. 

Again this is really good work. Thanks @ssd71 @wtrocki